proto/panacea/aol/v2/genesis.proto

```diff
@@ -7,7 +7,7 @@ import "panacea/aol/v2/writer.proto";
 import "panacea/aol/v2/topic.proto";
 
 option java_multiple_files = true;
-option go_package = "github.com/medibloc/panacea-core/x/aol/types";
+option go_package = "github.com/medibloc/panacea-core/v2/x/aol/types";
```

Comment:
I changed go_package to v2, the reason is to apply v2 to the package name even when importing the generated *.pb.go file.

Reply:
Example here(https://github.com/medibloc/panacea-core/pull/363/files#diff-4178f3a958cb7d27bee7d4cc8057249dfd825b9f2104cc1ee0cbc69c97e3942dR10)